api/v2/log.go

```diff
@@ -12,24 +12,25 @@ import (
 // is the value
 func (api *API) LogError(err error, message string, fields ...interface{}) func(c *gin.Context, code ...int) {
 	// create base entry
-	entry := api.l.WithFields(log.Fields{
-		"service": api.service,
-	})
-
-	// add additional fields
-	if fields != nil {
-		for i := 0; i < len(fields); i += 2 {
-			if i+1 < len(fields) {
-				entry = entry.WithField(fields[i].(string), fields[i+1])
-			}
-		}
-	}
+	logger := api.l
 
 	// write log
 	if err == nil {
-		entry.Error(message)
+		// mod 2 allows us to check the fields are even in length
```

Comment:
see above, remove this whole chunk to just be `logger.Error(message`

Reply:
done